cmake/mission_defaults.cmake

```diff
@@ -14,6 +14,7 @@ set(MISSION_CORE_MODULES
     "cfe-core" 
     "osal"
     "psp"
+    "msg"
```

Comment:
Have we evaluated the possibility that the single `msg` perhaps should actually be three:

 - ccsds spacepacket (base)
 - ccsds extended
 - cfs cmd/tlm

I reviewed the internal "extension" library and noted that it is effectively replacing the _entire_ `msg` module with a different one, but then selectively re-compiling and referring to some (but not all) source files provided by the msg module, as well as giving itself an include path to the private headers within this module.

The paradigm should be that external entities don't depend on private headers - nor should source files be compiled by a script that isn't version controlled with the said source files.

It would seem, however, that if it the "msg" module were actually three different modules, then only one would need to be replaced for modifying the secondary header, and the other two (ccsds + extended) can be used as-is.